docs/topics/auth/default.txt

```diff
@@ -762,8 +762,10 @@ Redirecting unauthorized requests in class-based views
 ------------------------------------------------------
 
 To ease the handling of access restrictions in :doc:`class-based views
-</ref/class-based-views/index>`, the ``AccessMixin`` can be used to redirect a
-user to the login page or issue an HTTP 403 Forbidden response.
+</ref/class-based-views/index>`, the ``AccessMixin`` can be used to configure
+the behavior of a view when the conditions are not met.  An authenticated user
+will always get an HTTP 403 Forbidden response.  An unauthenticated user can
```

Comment:
When I read "... user can be redirected..." it makes it sound like it is a matter of chance. How about adding something like "depending on configuration" to clarify?

Reply:
👍